packages/salesforcedx-vscode-replay-debugger/src/states/frameState.ts

```diff
@@ -0,0 +1,15 @@
+import { SFDC_TRIGGER } from '../constants';
+
+export class FrameState {
```

Comment:
Thoughts on making this an abstract class?

Reply:
No, only because once we start adding things to process frame names I'd like to be able to unit test it directly (albeit at that point I'd have to add getters for the protected members).